views/partials/menu.handlebars

```diff
@@ -0,0 +1,34 @@
+<!-- Navigation -->
```

Comment:
Good progress, let's finish this up! :)

Reply:
The admin page should be the same as all other pages, but the body is a form that allows the admin to create a blog post. Viewing the blog posts should be on a different page. What you show after the post is submitted on the admin page is up to you.